crates/ethcore/src/machine/impls.rs

```diff
@@ -265,59 +262,29 @@ impl EthereumMachine {
         &self,
         header: &mut Header,
         parent: &Header,
-        gas_floor_target: U256,
+        _gas_floor_target: U256,
         gas_ceil_target: U256,
     ) {
         header.set_difficulty(parent.difficulty().clone());
         let gas_limit = parent.gas_limit() * self.schedule(header.number()).eip1559_gas_limit_bump;
         assert!(!gas_limit.is_zero(), "Gas limit should be > 0");
 
+        header.set_gas_limit({
+            let bound_divisor = self.params().gas_limit_bound_divisor;
+            if gas_limit < gas_ceil_target {
+                cmp::min(gas_ceil_target, gas_limit + gas_limit / bound_divisor - 1)
+            } else {
+                cmp::max(gas_ceil_target, gas_limit - gas_limit / bound_divisor + 1)
+            }
+        });
```

Comment:
👍 should be fine

Reply:
It just should be communicated that for AuRa/Clique engines `gas_flooor_target` is replaced with `gas_ceil_target`.

Reply:
hm, maybe we can even use something like `gas_target = if eip1559 { gas_floor_target} else {gas_ceil_target}` :)

Reply:
@rakita How about the `--gas-floor-target` CLI option? Will it be renamed to `--gas-ceil-target`?

Reply:
@varasev `--gas-ceil-target` is currently `--gas-cap` option

Reply:
Ok, so the above changes with gas target won't affect the current xDai configs, right?

Reply:
> It just should be communicated that for AuRa/Clique engines `gas_flooor_target` is replaced with `gas_ceil_target`.

Yes, do you think that version changelog is not enough, should we announce it directly somewhere?

Reply:
@varasev it shouldn't, `gas_limit` is overwritten down the line with `blockGasLimitContract`.
@sunce86 changelog is fine.

Reply:
> hm, maybe we can even use something like `gas_target = if eip1559 { gas_floor_target} else {gas_ceil_target}` :)

Yes, accepted. We should introduce this change for ethash after 1559 activation. Fixed.

> Ok, so the above changes with gas target won't affect the current xDai configs, right?

Updating the configs is not mandatory, but behavior will definitely be different, so aligning the  --gas-floor-target and --gas-cap options with new behavior is desired.

Reply:
> @varasev it shouldn't, `gas_limit` is overwritten down the line with `blockGasLimitContract`.

But if we don't use `blockGasLimitContract` (it is not activated on xDai), the behavior will differ from Nethermind?

Reply:
with `gas_target = if eip1559 { gas_floor_target} else {gas_ceil_target}` it will be same even without `blockGasLimitContract`. And in general, setting different `gas_limit` should not cause consensus error.